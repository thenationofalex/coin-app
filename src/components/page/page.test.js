import React, {Component} from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Page from './index'
 
test('Page renders', () => {
  const wrapper = shallow(
    <Page />
  );
 
  expect(toJson(wrapper)).toMatchSnapshot()
});
