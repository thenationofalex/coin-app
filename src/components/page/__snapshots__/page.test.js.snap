// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<React.Fragment>
  <Header />
  <Content>
    <Connect(CoinList) />
  </Content>
</React.Fragment>
`;
